Cython/Utility/Builtins.c

```diff
@@ -303,6 +303,21 @@ static PyObject *__Pyx_PyLong_AbsNeg(PyObject *n) {
 #define __Pyx_PyNumber_Power2(a, b) PyNumber_Power(a, b, Py_None)
 
 
+//////////////////// divmod_int.proto //////////////////
+
+static CYTHON_INLINE PyObject* __Pyx_divmod_int(int a, int b){
+    PyObject* result_tuple;
+    PyObject* args;
+    div_t res = div(a, b);
+    args = PyTuple_Pack(2, res.quot, res.rem);
```

Comment:
Also don't think this is right - `PyTuple_Pack` takes `PyObject*` and you're passing it C `int`

Reply:
Oh, I mistakenly thought PyTuple_Pack() passes C arguments. Looks like I need to modify that to Py_BuildValue("(ii)", res.quot, res.rem).